Flutter/pdf-viewer/sticky-notes.md

```diff
@@ -0,0 +1,139 @@
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### Add sticky note annotation programmatically 
+
+You can create and add a sticky note annotation to a PDF document programmatically using the [addAnnotation](https://pub.dev/documentation/syncfusion_flutter_pdfviewer/latest/pdfviewer/PdfViewerController/addAnnotation.html) method of the [PdfViewerController](https://pub.dev/documentation/syncfusion_flutter_pdfviewer/latest/pdfviewer/PdfViewerController-class.html). The following example explains how to create a sticky note with a comment icon and add it to the first page of a PDF document.
+
+{% tabs %}
+{% highlight dart hl_lines="18" %}
+
+void addStickyNoteAnnotation() {
+  // Create a sticky note annotation
+  final StickyNoteAnnotation stickyNote = StickyNoteAnnotation(
+    // Set the page number for the sticky note annotation
+    pageNumber: 1,
+    // Set the text for the sticky note annotation
+    text: 'This is a sticky note annotation',
+    // Set the icon for the sticky note annotation
+    icon: PdfStickyNoteIcon.comment,
+    // Set the position for the sticky note annotation
+    position: const Offset(100, 100),
+  );
+
+  // Set the color for the sticky note annotation
+  stickyNote.color = Colors.yellow;
+
+  // Add the sticky note annotation to the PDF viewer
+  _pdfViewerController.addAnnotation(stickyNote);
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Sticky note annotation settings 
+
+In the sticky note annotation mode, the annotation will be added with a default appearance. You can modify the annotation after it has been added to the pages. However, if you need to define the appearance before adding sticky note annotations to the document, you can change its default settings using the [PdfViewerController.annotationSettings.stickyNote](https://pub.dev/documentation/syncfusion_flutter_pdfviewer/latest/pdfviewer/PdfAnnotationSettings/stickyNote.html). For that, you need to obtain the default sticky note annotation settings.
+
+The following example explains how to obtain the default sticky note annotation settings and modify some of their properties. Similarly, you can modify all the other properties.
+
+{% tabs %}
+{% highlight dart %}
+
+void customizeDefaultStickyNoteSettings() {
+  // Obtain the default sticky note annotation settings from the PdfViewerController instance.
+  PdfStickyNoteAnnotationSettings stickyNoteSettings =
+      _pdfViewerController.annotationSettings.stickyNote;
+
+  // Modify the default properties.
+  stickyNoteSettings.icon =
+      PdfStickyNoteIcon.comment; // Set the default icon to Comment.
+  stickyNoteSettings.color = Colors.yellow; //Stroke color
+  stickyNoteSettings.opacity = 0.75; // 75% Opacity
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Edit the selected sticky note annotation
+
+### Edit the text using UI interaction
+
+When you double-tap the selected sticky note, the text editor opens. Edit the text and click "Save" to modify the text.
```

Comment:
To opened Editor double tap is used in Flutter also?

<img width="559" alt="image" src="https://github.com/user-attachments/assets/43c12634-a6ac-42d1-84f5-d96606b8ea44">

Reply:
Yes